modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/commands/CliCommandTestInitializedIntegrationBase.java

```diff
@@ -31,10 +32,14 @@ public class CliCommandTestInitializedIntegrationBase extends CliCommandTestNotI
     @Override
     void beforeAll(TestInfo testInfo) {
         startNodes(testInfo);
-        super.initializeCluster(metaStorageNodeName(testInfo));
+        initializeCluster(metaStorageNodeName(testInfo));
     }
 
     protected String metaStorageNodeName(TestInfo testInfo) {
         return testNodeName(testInfo, 0);
     }
+
+    protected List<String> allNodeNames() {
+        return CLUSTER_NODE_NAMES;
+    }
```

Comment:
Why is it here? This static field in IgniteTestBase has protected modifier and you expand it by NON static protected method in some child class, absolutely don't understand what the reason

Reply:
Agreed to add the `static` modifier to the method signature.